src/vs/base/common/htmlContent.ts

```diff
@@ -201,3 +202,13 @@ export function parseHrefAndDimensions(href: string): { href: string; dimensions
 	}
 	return { href, dimensions };
 }
+
+export function markdownCommandLink(command: { title: string; id: string; arguments?: unknown[] }): string {
+	const uri = URI.from({
+		scheme: Schemas.command,
+		path: command.id,
+		query: command.arguments?.length ? encodeURIComponent(JSON.stringify(command.arguments)) : undefined,
+	}).toString();
+
+	return `[${command.title}](${uri})`;
```

Comment:
Just to be safe, you may want to escape the `title` using `escapeMarkdownSyntaxTokens`